.changeset/@mastra_deployer-9779-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@mastra/deployer": patch
+---
+dependencies updates:
+  - Updated dependency [`@rollup/plugin-alias@6.0.0` ↗︎](https://www.npmjs.com/package/@rollup/plugin-alias/v/6.0.0) (from `5.1.1`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)